backend/domain/Room.js

```diff
@@ -64,6 +79,28 @@ class Room {
         this.roundNumber++;
     }
 
+    getWinningRole() {
+        const numOfMafiaAlive = this.getPlayersByRole(roles.MAFIA).filter((player) => player.isAlive);
+        const numOfJesterAlive = this.getPlayersByRole(roles.JESTER).filter((player) => player.isAlive);
+        const numOfCiviliansAlive = [
+            this.getPlayersByRole(roles.CIVILIAN),
```

Comment:
I dont think this will work. Did you forget to spread (...) the list returned from `this.getPlayersByRole()`.  

I think currently, it will go through each list and check if list.isAlive is true?

I could be wrong if u wrote tests for this

Reply:
fixed